src/product.hpp

```diff
@@ -5,50 +5,23 @@
  * Pin Definitions
  *****************************************************************************/
 /**
- * USB Power Detection Pin
+ * USB Power Detection Pin TODO
  */
 #define SF_USB_PWR_DETECT_PIN   45
 /**
- * Water Detect Enable Pin
+ * Battery Charging Status Pin 
  */
-#define WATER_DETECT_EN_PIN   B4
-/**
- * Water Detect Pin
- */
-#define WATER_DETECT_PIN      B5
-/**
- * @brief Manufacturing Water Detect Pin
- * 
- */
-#define WATER_MFG_TEST_EN     A3
-/**
- * Battery Charging Status Pin
- */
-#define STAT_PIN              A5
+#define STAT_PIN              50
```

Comment:
Is this correct?

Reply:
50 looks to be correct, marked as "PMIC charge status. Can be connected to an LED. Active low." on the specs